Configuration/PyReleaseValidation/python/relval_2026.py

```diff
@@ -19,15 +19,15 @@
 numWFIB.extend([23434.99,23434.999]) #2026D49 premixing combined stage1+stage2 (ttbar+PU200, ttbar+PU50 for PR test)
 numWFIB.extend([23234.21,23434.21]) #2026D49 prodlike, prodlike PU
 numWFIB.extend([23234.103]) #2026D49 aging
-numWFIB.extend([28234.0]) #2026D60
-numWFIB.extend([29834.0]) #2026D64
-numWFIB.extend([30234.0]) #2026D65
-numWFIB.extend([30634.0]) #2026D66
-numWFIB.extend([31034.0]) #2026D67
-numWFIB.extend([31434.0]) #2026D68
-numWFIB.extend([31834.0]) #2026D69
-numWFIB.extend([32234.0]) #2026D70
-numWFIB.extend([32634.0]) #2026D71
+numWFIB.extend([24434.0]) #2026D60
```

Comment:
workflow numbers are not allowed to change (in general). please update https://github.com/cms-sw/cmssw/blob/e3a2e63fcfb011c2b053e921dd088e0cc3d9e1b1/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L62 with the range of indices from the removed D53 workflows, so that the other workflow numbers stay consistent.

Reply:
@kpedro88 you are right, it would be confusing (I fixed the comments but changing the workflow numbers is confusing, I agree)